{"name": "Проверка ошибки при попытке удаления курьера с несуществующим id", "status": "passed", "steps": [{"name": "Удаление курьера", "status": "passed", "parameters": [{"name": "courier_id", "value": "'234563256'"}], "start": 1747299820169, "stop": 1747299820252}], "start": 1747299820168, "stop": 1747299820254, "uuid": "07571e8d-dfa9-49d5-b3b3-fdfb5e58f7b5", "historyId": "3f588d613e3f1bdea1b2b2aa5c05c8e5", "testCaseId": "3f588d613e3f1bdea1b2b2aa5c05c8e5", "fullName": "tests.test_couriers.test_courier_delete.TestCourierDelete#test_courier_delete_error_with_empty", "labels": [{"name": "parentSuite", "value": "tests.test_couriers"}, {"name": "suite", "value": "test_courier_delete"}, {"name": "subSuite", "value": "TestCourierDelete"}, {"name": "host", "value": "DESKTOP-VE6DRBG"}, {"name": "thread", "value": "8232-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_couriers.test_courier_delete"}]}